corpus_raw.conllulex

```diff
@@ -19902,7 +19902,7 @@
 5	to	to	PART	TO	_	6	mark	_	_	_	_	_	`i	`i	_	_	_	_
 6	say	say	VERB	VB	_	4	xcomp	_	_	_	_	_	_	_	_	_	_	_
 7	that	that	PRON	DT	_	6	obj	_	_	_	_	_	_	_	_	_	_	_
-8	down	down	ADP	RP	_	10	compound:prt	_	_	_	_	_	_	_	_	_	_	_
+8	down	down	ADV	RB	_	10	compound:prt	_	_	_	_	_	_	_	_	_	_	_
```

Comment:
Not `compound:prt`. `advmod`?

Reply:
Right it's not `compound:prt`, but I've generally not been touching the parses since they're not gold anyway. Should we still make scattered manual corrections anyway?

Reply:
(The reason I corrected POS tags here though is that the validator complains about ADP without supersenses)

Reply:
Got it. If we're manually correcting the tag I think it's worth correcting the dependencies too for consistency. I did some spot checks on the UD in the latest commits to the dev branch. But we're obviously not fully reviewing the trees.